src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -803,6 +803,10 @@ private RuntimeConstructorInfo[] PopulateConstructors(Filter filter)
                     return list.ToArray();
                 }
 
+                [UnconditionalSuppressMessage ("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
```

Comment:
I've submitted https://github.com/dotnet/runtime/pull/52535 to see whether we can just delete the problematic code instead.

I think this would only do something if we allowed instance fields on interfaces, but that smells a lot like multiple-inheritance problems, so I don't think we're ever going to allow that.

Reply:
Thanks a lot for testing this.